activemodel/lib/active_model/validations/acceptance.rb

```diff
@@ -20,6 +20,11 @@ def setup!(klass)
         klass.send(:attr_reader, *attr_readers)
         klass.send(:attr_writer, *attr_writers)
       end
+
+      def valid_option?(value)
+        accept_options = [options[:accept]].flatten
+        accept_options.include?(value)
```

Comment:
To preserve backward compatibility for anyone who currently uses "accept: 'I agree.'"

```bash
  1 || ^[[H^[[JRun options: --seed 14171                                                                                                                               
  2 ||.
  3 || # Running:
  4 ||.
  5 || ...F..
  6 ||.
  7 || Finished in 0.040334s, 148.7579 runs/s, 322.3087 assertions/s.
  8 ||.
  9 ||   1) Failure:
 10 || AcceptanceValidationTest#test_terms_of_service_agreement_with_accept_value [test/cases/validations/acceptance_validation_test.rb:47]:
 11 || Failed assertion, no message given.
 12 ||.
 13 || 6 runs, 13 assertions, 1 failures, 0 errors, 0 skips
```

```ruby
["I agree."].flatten # ["I agree."]
[[1, "true"]].flatten # [1, "true"]
```

If it's overkill please let me know, I'll gladly update.